app/mailers/request_mailer.rb

```diff
@@ -0,0 +1,24 @@
+class RequestMailer < ApplicationMailer
+  def request_cancel_partner_notification(request_id:)
+    @request ||= Request.find(request_id)
+    @organization ||= @request.organization
+    @partner = @request.partner
+
+    # Generate the request_items in a formatted way so
+    # I can render Item names
+    item_ids = @request.request_items.map { |r| r['item_id'] }
+    items = Item.where(id: item_ids)
+    @formatted_requested_items = @request.request_items.map do |rt|
+      {
+        name: items.find { |i| i.id == rt['item_id'] }&.name,
```

Comment:
is the `&.` because the request might be referencing an item id that doesn't exist in the db?

Reply:
@albertchae, unfortunately, yeh T_T. We'll need to work on building better data integrity eventually.